core/tests/protractor_desktop/topicsAndSkillsDashboard.js

```diff
@@ -57,10 +54,42 @@ describe('Topics and skills dashboard functionality', function() {
 
   it('should add a new topic to list', function() {
```

Comment:
The test seems to be testing more than what is stated in the description. Have the description actually match the test in question. You can split this test into two separate ones, if you like.

Reply:
Fixed.